web/src/components/pages/dashboard/stats/stats-card.tsx

```diff
@@ -17,6 +21,7 @@ export default function StatsCard({
   scrollable,
   currentLocale,
 }: {
+  id?: string;
```

Comment:
Thinking maybe this should be `storageKey`? Matches the other usage of it. Doesn't make much sense for the API of StatsCard though. A unique ID is what it is for StatsCard...